client/src/containers/TodoCardList/Item/index.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import TodoCard from 'src/components/TodoCardList/Item'
+import api from './requests'
+
+class TodoCardContainer extends Component {
+  state = {
+    readiness: this.props.readiness || false,
+  }
+
+  render = () => (
+    <TodoCard
+      description={this.props.description}
+      readiness={this.state.readiness}
+      onChange={this.updateReadiness}
+    />
+  );
+
+  updateReadiness = (event) => {
+    this.setState({ readiness: event.target.checked })
```

Comment:
Looks like we update state before making the request, that means that we use optimistic UI here. In order to use it correctly, we have to set up validations, kind of (that's needed to keep server and client consistent, avoid situation when on client item is shown as completed, but the request to update failed):

```
- update state
- send the request
  - success: do nothing 
  - error: revert state changes
```

We can go with simpler way for now - pessimistic UI:
```
- send the request
  - success: update state 
  - error: show alert
```

```js
    api.card.update(
      { id: this.props.id, readiness: event.target.checked },
      (response) => {
        this.setState({ readiness: reponse.card.readiness })
      },
      (error) => {
        alert(error)
      },
    )
```